### PR TITLE
docs(ts): fix vtkProp typescript definitions

### LINKED ISSUE #2317: Bugs in TypeScript type definitions.
### High-level description

2 bugs in typescript type definitions.

### Steps to reproduce

Try to use vtkFullScreenRenderWindow from a typescript project.

### Detailed behavior

* `rootContainer` is not defined on IFullScreenRenderWindowInitialValues
* Param to `setRendertimemultiplier` needs a non-any type.

```
~$ yarn type
yarn run v1.22.15
$ vue-tsc -p tsconfig.check.json --noEmit
src/components/MeshViewer.vue:30:72 - error TS2345: Argument of type '{ rootContainer: HTMLElement; }' is not assignable to parameter of type 'IFullScreenRenderWindowInitialValues'.
  Object literal may only specify known properties, and 'rootContainer' does not exist in type 'IFullScreenRenderWindowInitialValues'.

30     const fullScreenRenderer = vtkFullScreenRenderWindow.newInstance({ rootContainer });
                                                                          ~~~~~~~~~~~~~

node_modules/@kitware/vtk.js/Rendering/Core/Prop.d.ts:235:29 - error TS7006: Parameter 'renderTimeMultiplier' implicitly has an 'any' type.

235     setRendertimemultiplier(renderTimeMultiplier): boolean;
                                ~~~~~~~~~~~~~~~~~~~~
```

### Expected behavior

Type checking should pass (without having to disable library type checks).

### Environment
- **vtk.js**: 23.0.1
- **OS**: n/a
- **Browser**: n/a

### PR DESCRIPTION
**Pull request by daker**
fix #2317

### Changes

- [x] Documentation and TypeScript definitions were updated to match those changes

### PR and Code Checklist

- [x] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [x] Run `npm run reformat` to have correctly formatted code

### Testing

- [ ] This change adds or fixes unit tests 
- [ ] Tested environment:
  - **vtk.js**: 
  - **OS**: 
  - **Browser**:

### CODE REVIEW

File: Sources/Rendering/Misc/FullScreenRenderWindow/index.d.ts [hunk 1]
```diff
@@ -13,11 +13,13 @@ import vtkRenderWindowInteractor from "../../Core/RenderWindowInteractor";
  */
 export interface IFullScreenRenderWindowInitialValues {
 	background?: RGBColor;
+	container?: HTMLElement
```
[the remark this answers is not in the document]
Comment (jourdain) [reply Sources/Rendering/Misc/FullScreenRenderWindow/index.d.ts:16]: I kind of agree, I don't want users to rely on `FullScreenRenderWindow` when that are not full screen.

**Comment by daker** [comment]
@floryst i have fixed the vtkProp issue if you want to have a look

**Comment by floryst** [comment]
LGTM, thanks!

**Comment by floryst** [comment]
LGTM, thanks!

**Pull request merged**